src/app.js

```diff
@@ -0,0 +1,62 @@
+import isYes from '../src/is-yes.js';
+
+const quizButton = document.getElementById('quiz-button');
+const result = document.getElementById('result');
+
+quizButton.onclick = function() {
+    //console.log('working??');
```

Comment:
In the future, please delete any console logs and commented out code before submitting